dist/index.html

```diff
@@ -10,42 +10,19 @@
 <body>
   <div class="h1-container">
     <h1>Travel Tracker</h1>
+    <p id="welcomeTravelerMsg" class="h1-welcom-msg"></p>
     <img class="h1-user-icon" src="images/user-icon.svg" alt="black user-icon">
   </div>
   <nav class="nav-container">
-    <button class="nav-button" id="addNewTrip">+Add new trip</button>
-    <button class="nav-button" id="pastTrips">Past trips</button>
-    <button class="nav-button" id="presentTrips">Present trips</button>
-    <button class="nav-button" id="upcomingTrips">Upcoming trips</button>
-    <button class="nav-button" id="pendingTrips">Pending trips</button>
-    <button class="nav-button" id="travelExpenses">Travel Expenses</button>
+    <button class="nav-button" id="addNewTripButton">+Add new trip</button>
```

Comment:
Are there any accessibility issues to consider for 'button' elements?  I may be wrong but I thought one of our accessibility lessons from class touched on buttons.  Maybe something to consider?